packages/@n8n/db/src/repositories/workflow-statistics.repository.ts

```diff
@@ -55,59 +55,59 @@ export class WorkflowStatisticsRepository extends Repository<WorkflowStatistics>
 		isRootExecution: boolean,
 	): Promise<StatisticsUpsertResult> {
 		const dbType = this.globalConfig.database.type;
-		const { tableName } = this.metadata;
+		const escapedTableName = this.manager.connection.driver.escape(this.metadata.tableName);
+
 		try {
 			if (dbType === 'sqlite') {
 				await this.query(
-					`INSERT INTO "${tableName}" ("count", "rootCount", "name", "workflowId", "latestEvent")
-					VALUES (1, ${isRootExecution ? '1' : '0'}, "${eventName}", "${workflowId}", CURRENT_TIMESTAMP)
+					`INSERT INTO ${escapedTableName} ("count", "rootCount", "name", "workflowId", "latestEvent")
```

Comment:
Maybe there's a reason, but curious if this really needs to be a raw query, one per DB, instead of native `typeorm` methods?